.github/copilot-instructions.md

```diff
@@ -1,92 +1,125 @@
+- **Lint**: `bundle exec rubocop` (handled by CI, don't suggest fixes)
+- **Server**: `foreman start -m all=1`
+- **Console**: `bundle exec rails console`
+- **DB**: `make db` (setup + migrate)
+
+### Quick Reference
+- **Config files**: `modules/[name]/config/` or main `config/`
+- **Serializers**: End controllers with `render json: object, serializer: SomeSerializer`
+- **Auth**: Most endpoints need `before_action :authenticate_user!`
+- **Jobs**: Use `perform_async` for background work, `perform_in` for delayed
+- **Flipper in tests**: Never use `Flipper.enable/disable` - always stub with `allow(Flipper).to receive(:enabled?).with(:feature).and_return(true)`
+
+### Common Patterns
+- Controllers in `modules/[name]/app/controllers` or `app/controllers`
+- Background jobs in `app/sidekiq/` - use for operations >2 seconds
+- External service clients in `lib/` with Faraday + timeouts + retries
+- Feature flags via Flipper - always wrap debugging logs
+- Strong parameters required - never use `params` directly
+- Error responses use envelope: `{ error: { code, message } }`
+- Service objects return `{ data: result, error: nil }` pattern
+
+### Key Dependencies
+- PostGIS required for database
+- Sidekiq Enterprise (may need license)
+- VCR cassettes for external service tests
+- Settings: `config/settings.yml` (alphabetical order required)
+
+### VA Service Integration
+- **BGS**: Benefits data, often slow/unreliable
+- **MVI**: Veteran identity, use ICN for lookups
+- **Lighthouse**: Preferred over legacy EVSS
+- **EVSS**: Being deprecated, migrate to Lighthouse
+
+## For PR Reviews - Human Judgment Issues
+
+### ⚠️ NO DUPLICATE COMMENTS - Consolidate Similar Issues
+
+### Security & Privacy Concerns
+- **PII in logs**: Check for email, SSN, medical data in log statements
+- **Hardcoded secrets**: API keys, tokens in source code
+- **Missing authentication**: Controllers handling sensitive data without auth checks
+- **Mass assignment**: Direct use of params hash without strong parameters
+
+### Business Logic Issues
+- **Non-idempotent operations**: Creates without duplicate protection
+- **Blocking operations in controllers**: sleep(), long external calls, file processing
+- **Wrong error response format**: Not using VA.gov standard error envelope
+- **Service method contracts**: Returning `{ success: true }` instead of data/error pattern
+
+### VA-Specific Patterns
+- **New logging without Flipper**: Suggest wrapping debug logs with feature flags
+- **External service calls**: Missing timeouts, retries, or error handling context
+- **Background job candidates**: Synchronous operations that should be async
+- **Feature flag testing**: Using .enable/.disable instead of stubbing in tests
+- **ICN vs User ID**: Use ICN for veteran lookups with MVI/BGS
+- **Form serialization**: Use form objects for complex form submissions
+- **Lighthouse migration**: Flag EVSS usage, suggest Lighthouse alternatives
```

Comment:
I don't think this is needed anymore since all of the migration has already completed.